index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Should I bring a jacket?</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fredoka+One&family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" type="text/css" href="styles/index.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Should I bring a jacket 🧥?</h1>
+      <span>
+        displaying for the city of
+        <span id="cityNameHeader"></span>
+      </span>
+    </header>
+    <main>
+      <section class="control-section">
+        <div class="control-box">
+          <h2>Temperature</h2>
+          <div class="display-flex">
+            <div class="temperature-control">
+              <span id="increaseTempButton">⬆️</span>
+              <span id="decreaseTempButton">⬇️</span>
+            </div>
+            <span id="tempValue"></span>
+          </div>
+        </div>
```

Comment:
whoops try not to use `div`! There's several other semantic html tags we can use like `section`, `main`, `article`, etc. instead